service/frontend/workflow_handler.go

```diff
@@ -35,6 +35,15 @@ import (
 	"time"
 
 	"github.com/pborman/uuid"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/health"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/status"
```

Comment:
Could you double check your IDE settings? We want to avoid imports moving around every time someone touches a file

Reply:
In a previous PR, it was noted that temporal imports should be separated from other imports. It would seem the pragmatic choice to fix offending import ordering as we move along. Is the updated ordering incorrect?

Reply:
3 groups, all sorted internally: std, 3rd party, `go.temporal.io/server/*`. This is GoLand settings:
![image](https://github.com/user-attachments/assets/db294e0e-727d-4ca1-902c-e70b7c7432ec)